library/src/main/java/com/novoda/downloadmanager/lib/DownloadInfo.java

```diff
@@ -440,12 +440,13 @@ private NetworkState checkSizeAllowedForNetwork(int networkType) {
      * create a {DownloadThread} and enqueue it into given
      * {@link Executor}.
      *
-     * @param collatedDownloadInfo
```

Comment:
I delete them when nobody has written javadoc about the parameter, if it's undocumented whats the point in it?